app/src/main/java/com/github/gogetters/letsgo/ChatActivity.kt

```diff
@@ -0,0 +1,57 @@
+package com.github.gogetters.letsgo
+
+import android.os.Bundle
+import android.os.Message
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.ArrayAdapter
+import android.widget.EditText
+import android.widget.ListView
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.github.gogetters.letsgo.chat.ChatMessage
+import com.github.gogetters.letsgo.chat.MessageAdapter
+import com.google.firebase.auth.FirebaseAuth
+import java.util.*
+
+
+class ChatActivity : AppCompatActivity() {
+    // the text field to which messages are written
+    private lateinit var entryText: EditText
+    private lateinit var adapter: MessageAdapter
+    private val DEFAULT_USERNAME = "Opponent"
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_chat)
+
+        entryText = findViewById(R.id.editText)
```

Comment:
not sure what the "best practices" for this are, but I feel like we should have a sort of "standard" for all of the IDs. For example, one thing I saw was to make them all start with the name of the activity (in this case chatEditText for example)